src/azure-cli/azure/cli/command_modules/container/_params.py

```diff
@@ -116,6 +128,14 @@ def load_arguments(self, _):
         c.argument('log_analytics_workspace', help='The Log Analytics workspace name or id. Use the current subscription or use --subscription flag to set the desired subscription.')
         c.argument('log_analytics_workspace_key', help='The Log Analytics workspace key.')
 
+    with self.argument_context('container create', arg_group='Container Group Profile Reference') as c:
+        c.argument('container_group_profile_id', help='The reference container group profile ARM resource id.')
+        c.argument('container_group_profile_revision', help='The reference container group profile revision.')
+
+    with self.argument_context('container create', arg_group='Standby Pool Profile') as c:
+        c.argument('standby_pool_profile_id', help='The standby pool profile ARM resource id from which the container will be reused.')
+        c.argument('fail_container_group_create_on_reuse_failure', help='The flag indicating whether to fail the container group creation if the standby pool reuse failed.', action='store_true')
```

Comment:
It is better to add a shorter option to make it more easier for users to use this param
```suggestion
        c.argument('fail_container_group_create_on_reuse_failure', help='The flag indicating whether to fail the container group creation if the standby pool reuse failed.', action='store_true')
```